character_controller_test.tscn

```diff
@@ -1733,19 +1733,19 @@ script = ExtResource("3_tib08")
 entity = NodePath("..")
 friendly = true
 
-[node name="Polygon2D" type="Polygon2D" parent="Hurtbox"]
-position = Vector2(-106.466, -200.732)
-scale = Vector2(0.6195, 0.487805)
-texture = ExtResource("4_380wk")
-polygon = PackedVector2Array(157, 391, 156.463, 431.973, 188, 432, 187.755, 391.156)
-uv = PackedVector2Array(160.544, 399.32, 156.463, 431.973, 187.075, 429.932, 187.755, 391.156)
-
 [node name="CollisionShape2D" type="CollisionShape2D" parent="Hurtbox"]
 shape = SubResource("RectangleShape2D_sbxtj")
 
 [node name="HurtboxState" type="Node" parent="Hurtbox"]
 script = ExtResource("4_1c3yk")
 
+[node name="Sprite2D" type="Sprite2D" parent="Hurtbox"]
+position = Vector2(1.41799, -0.401596)
+scale = Vector2(0.285608, 0.28652)
+texture = ExtResource("4_380wk")
```

Comment:
Somehow became `4_ppaue` instead

Reply:
To make things even more fun, in the modified copy (not original), this resource is now `4_itlr7` instead.